openlibrary/plugins/inside/code.py

```diff
@@ -65,9 +70,7 @@ def inside_search_select(params):
                 return {'error': 'Error converting search engine data to JSON'}
 
         def quote_snippet(snippet):
-            trans = { '\n': ' ', '{{{': '<b>', '}}}': '</b>', }
-            re_trans = re.compile(r'(\n|\{\{\{|\}\}\})')
-            return re_trans.sub(lambda m: trans[m.group(1)], web.htmlquote(snippet))
+            return self.quote_snippet(snippet)
```

Comment:
Just curious. Why create this pass-through instead of just passing `self.quote_snippet` to `render_template()` down below?